concordia/admin/actions.py

```diff
@@ -1,9 +1,25 @@
+import uuid
+
 from django.contrib import messages
 from django.utils.timezone import now
 
 from ..models import Asset, Transcription, TranscriptionStatus
 
 
+def anonymize_action(modeladmin, request, queryset):
+    count = queryset.count()
+    for user_account in queryset:
+        user_account.username = uuid.uuid4()
+        user_account.email = ""
+        user_account.password = ""
```

Comment:
Can we change this to call [`set_unusable_password()`](https://docs.djangoproject.com/en/2.2/ref/contrib/auth/#django.contrib.auth.models.User.set_unusable_password)?
```suggestion
        user_account.set_unusable_password()
```